docs/05-infrastructure-nodes.md

```diff
@@ -331,8 +337,8 @@ Now that you have some special nodes, it's time to move various
 infrastructure components onto them.
 
 ### NOTE
-The following assumes you used version `0.11.0` of the installer. This would
-give you a `clusterversion` of `4.0.0-0.2`.
+The following assumes you used version `0.13.1` of the installer. This would
+give you a `clusterversion` of `4.0.0-0.6`.
```

Comment:
Installer version and clusterversion is updated to **0.14.0** and **4.0.0-0.7**

http://post-office.corp.redhat.com/archives/aos-devel/2019-March/msg00113.html

Reply:
Fixed with 84fb55d